Modix/Modules/ModerationModule.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Discord;
+using Discord.Commands;
+
+using Modix.Data.Models.Moderation;
+using Modix.Services.Moderation;
+
+namespace Modix.Modules
+{
+    [Name("Moderation")]
+    [Summary("Guild moderation commands")]
+    public class ModerationModule : ModuleBase
+    {
+        public ModerationModule(IModerationService moderationService)
+        {
+            ModerationService = moderationService ?? throw new ArgumentNullException(nameof(moderationService));
+        }
+
+        [Command("note")]
+        [Summary("Applies a note to a user's infraction history.")]
+        public Task Note(
+            [Summary("The user to which the note is being applied.")]
+                IGuildUser subject,
```

Comment:
No, that's intentional. It's consistent with the way every other method in that class is written.

Attributes should be one-per-line. That keeps each single line to one logical thought, AND allows for the ever so tiny benefit of being able to easily comment one out, for debug/testing.

Attributes should also be on a line separate from the thing they're attached to. Again, one logical thought per line. Plus it would look really weird if a param had 3 attributes, then the param itself followed on the same line as the third one.

The extra indenting helps to visually distinguish separate parameters from each other.